crates/bdk/tests/wallet.rs

```diff
@@ -2298,7 +2322,7 @@ fn test_fee_amount_negative_drain_val() {
     let send_to = Address::from_str("tb1ql7w62elx9ucw4pj5lgw4l028hmuw80sndtntxt")
         .unwrap()
         .assume_checked();
-    let fee_rate = FeeRate::from_sat_per_vb(2.01);
+    let fee_rate = FeeRate::from_sat_per_kwu(500);
```

Comment:
Again, I am trying to follow the math:

`dbg!(psbt.clone().extract_tx().weight().to_wu() as f64 * 2.01)` gives `659.28`